feature-about/build.gradle.kts

```diff
@@ -10,18 +10,20 @@ dependencies {
     implementation(projects.coreModel)
     testImplementation(projects.coreTesting)
 
-    implementation(libs.kermit)
-    implementation(libs.hiltNavigationCompose)
+    implementation(libs.accompanistPager)
```

Comment:
I sort this alphabetically so it is easier to understand which one is which 😉

Reply:
I thought sorting dependencies is not so good idea. But I think it is better than I thought 👍 
May I ask how you are sorting them?

Reply:
I use IdeaVim to sort this 😄 

https://vim.fandom.com/wiki/Sort_lines